<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerWindow</class>
 <widget class="QMainWindow" name="PlayerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>–ü–ª–µ–µ—Ä</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QWidget {
    background-color: #1e001f;
    color: #ffcce6;
    font-family: 'Segoe UI', sans-serif;
}
QLabel {
    color: #ffb6d9;
}
QPushButton {
    background-color: #cc2b85;
    color: white;
    border-radius: 8px;
    padding: 6px 12px;
    font-weight: bold;
    border: none;
}
QPushButton:hover {
    background-color: #e0439a;
}
QSlider::groove:horizontal {
    height: 6px;
    background: #ff99cc;
    border-radius: 3px;
}
QSlider::handle:horizontal {
    background: white;
    border: 1px solid #ff66a3;
    width: 14px;
    margin: -5px 0;
    border-radius: 7px;
}
   </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainLayout">
    <property name="spacing">
     <number>16</number>
    </property>
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <item>
     <widget class="QLabel" name="labelTitle">
      <property name="styleSheet">
       <string notr="true">font-size: 18pt; font-weight: bold;</string>
      </property>
      <property name="text">
       <string>üéµ –ù–∞–∑–≤–∞–Ω–∏–µ —Ç—Ä–µ–∫–∞</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelArtist">
      <property name="text">
       <string>–ò—Å–ø–æ–ª–Ω–∏—Ç–µ–ª—å</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelCover">
      <property name="minimumSize">
       <size>
        <width>220</width>
        <height>220</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid #ff99cc; border-radius: 10px;</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="sliderProgress">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="layoutTimes">
      <item>
       <widget class="QLabel" name="labelCurrentTime">
        <property name="text">
         <string>0:00</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="spacerMid">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="labelDuration">
        <property name="text">
         <string>0:00</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="controlsLayout">
      <item>
       <widget class="QPushButton" name="btnPrev">
        <property name="icon">
         <iconset theme="QIcon::ThemeIcon::GoPrevious"/>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnStop">
        <property name="icon">
         <iconset theme="QIcon::ThemeIcon::MediaPlaybackStop"/>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnPlay">
        <property name="icon">
         <iconset theme="QIcon::ThemeIcon::MediaPlaybackStart"/>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnPause">
        <property name="icon">
         <iconset theme="QIcon::ThemeIcon::MediaPlaybackPause"/>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnNext">
        <property name="icon">
         <iconset theme="QIcon::ThemeIcon::MediaSkipForward"/>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
